packages/ui/src/molecules/SearchInput/SearchInput.test.tsx

```diff
@@ -6,18 +6,24 @@ import type { SearchInputProps } from './SearchInput'
 import SearchInput from './SearchInput'
 
 const Wrapper = (props: Partial<SearchInputProps>) => (
-  <SearchInput {...props} onSubmit={(value) => value} testId="search-input" />
+  <SearchInput testId="search-input" onSubmit={(value) => value} {...props} />
 )
 
 describe('SearchInput', () => {
-  it('`data-fs-search-input` is present', () => {
+  it('`data-fs-search-input-form` is present', () => {
     const { getByTestId } = render(<Wrapper />)
 
     expect(getByTestId('search-input')).toHaveAttribute(
-      'data-fs-search-input'
+      'data-fs-search-input-form'
     )
   })
 
+  it('`data-fs-search-input` is present and applied to `Input`', () => {
+    const { getByTestId } = render(<Wrapper data-fs-search-input />)
+
+    expect(getByTestId('store-input')).toHaveAttribute('data-fs-search-input')
```

Comment:
do we need to update here as well?

Reply:
No, we don't. This test is specifically to ensure that custom data-attributes will be applied to the right component, in this case, applied to the `Input`

Reply:
Nice! let's go ahead!